userspace/openpilot_python_dependencies.sh

```diff
@@ -2,11 +2,9 @@
 
 echo "installing uv..."
 
-export XDG_DATA_HOME="/usr/local"
-export CARGO_HOME="$XDG_DATA_HOME/.cargo"
+export XDG_DATA_HOME="/usr/local" # uv is installed in $XDG_DATA_HOME/../bin since 0.5.0
```

Comment:
removing the comment since it only makes sense in the context of this PR
```suggestion
export XDG_DATA_HOME="/usr/local" 
```